packages/frontend-bo/src/components/utils/DisplayInput.vue

```diff
@@ -72,7 +72,10 @@ const props = defineProps({
 });
 
 const rows = computed(() => {
-  return props.value.map((d) => {
+  const data = props.input.filter
+    ? props.input.filter(props.value)
+    : props.value;
```

Comment:
```suggestion
  const data = props.input?.filter(props.value) ?? props.value
```

Si tu veux plus one liner , a mon avis ca ca marche (a test, et ne l fais pas si tu n'aimes pas)

Reply:
ca ne marchait pas, sans que je comprenne trop pourquoi.
mais ca ca passe :
  const data =
    (props.input.filter && props.input.filter(props.value)) ?? props.value;